package.json

```diff
@@ -15,8 +15,8 @@
     "format": "prettier --write 'src/**/*.ts?(x)' && npm run lint -- --fix",
     "typecheck": "tsc --noEmit",
     "clean": "rm -rf ./dist ./tmp",
-    "build": "npm run clean && rollup --config && mv tmp/*.js dist",
-    "prepack": "npm run build",
+    "build": "npm run clean && rollup --config && mv tmp/*.js dist && attw --pack .",
+    "prepublishOnly": "npm run build",
```

Comment:
Moved to prepublishOnly, since this only runs on publish instead of on pack.

attw --pack calls npm pack, so you end up in a nice infinite loop.